docs/src/pages/docs/installation/installing_scratch.mdx

```diff
@@ -129,7 +129,7 @@ superset db upgrade
 Finish installing by running through the following commands:
 
 ```
-# Create an admin user (you will be prompted to set a username, first and last name before setting a password)
```

Comment:
[This content](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#flask-server) in the github repo and [this content](https://superset.apache.org/docs/installation/installing-superset-from-scratch#installing-and-initializing-superset) in the documentation website were mismatched. The info in the github repo is more useful. Hence syncing the docs.